### DISCUSSION TITLE
Nitrous oxide emissions in v1.1 dataset

### DISCUSSION THREAD

**Discussion by aoifecalnan** [USEPA/supply-chain-factors#20] [Q&A] [answered]
From looking at the Supply Chain Factors Dataset v1.1, I noticed the nitrous oxide emissions make a large difference to the impact of some emission factors, particularly those in the agricultural sector. All factors seem to be rounded to 3 decimal places. Is it possible this rounding could cause the nitrous oxide, and other emissions, to overestimate the CO2e impact when calculated?

**Comment by WesIngwersen** [comment, marked as answer]
@aoifecalnan, this is a very good point. Because N2O emissions are small in quantity, and you are correct that we provide only 3 digits because they decimal, then often the N2O emission factors are only showing 1 significant digit. This can present an issue because of N2O being a powerful GHG.  I am going to create an issue for the discussion for us to consider not just a strict 3 decimal place rounding for all factors.
